src/app.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+import AppBar from '@material-ui/core/AppBar'
+import ToolBar from '@material-ui/core/Toolbar'
+import Typography from '@material-ui/core/Typography'
+import Uploader from './uploader'
+import Load from './load'
+import CarCard from './car-card'
+import Cssbaseline from '@material-ui/core/CssBaseline'
+import MuiThemeProvider from '@material-ui/core/styles/MuiThemeProvider'
+import { createMuiTheme } from '@material-ui/core/styles'
+import blue from '@material-ui/core/colors/blue'
+import orange from '@material-ui/core/colors/orange'
+
+const theme = createMuiTheme({
+  palette: {
+    primary: blue,
+    secondary: orange
+  },
+  background: {
+    default: '#E0E0E0'
+  }
+})
+
+const styles = {
```

Comment:
Material UI has a built-in `withStyles` mechanism. Use that. Avoid doing things differently than your primary tools allow.